src/Validation/ItValidation.php

```diff
@@ -88,6 +88,48 @@ public static function cf($check)
         return (chr($sum % 26 + ord('A')) == $check[15]);
     }
 
+    
+	
+    /**
+     * Checks Partita IVA, VAT number for Italy.
+     *
+     * @param string $check The value to check.
+     * @return bool Success.
+     */
+	public static function piva($check){
```

Comment:
Shouldnt this have the same function name as the interface?

Reply:
VAT Number isn't present in the interface. I'm just following the file structure (as for the function cf)

Reply:
OK